syntax = "proto3";

message response_SE {
  string Result = 1;
  string State = 2;
}

